shopping-cart/src/pages/CartPage/CartPage.jsx

```diff
@@ -0,0 +1,280 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import API from '../../API';
+import './CartPage.sass'
+
+// #### 장바구니
+// ​
+// - [x] 해당 상품의 수량을 변경할 수 있다.
+//   - [x] 상품의 수량은 항상 1이상, 20이하여야 한다
+//     - [x] 상품의 수량이 1이면 상품 수량 감소할 수 없다.
+//     - [x] 상품의 수량이 20이면 상품 수량 증가할 수 없다.
+//   - [x] 해당 상품의 총 금액이 변경된다.
+//   - [x] 해당 상품이 체크되어있으면, 결제예상금액도 변경된다.
+// - [x] 모두선택 버튼이 체크되면, 상품들이 모두 선택된다.
+//   - [x] 모두선택 버튼이 체크가 풀리면, 상품들의 선택이 모두 해제된다.
+// - [x] 상품 삭제 버튼을 누르면, confirm 메시지가 보여진다.
+//   - [x] 확인을 누르면, 선택된 상품이 모두 삭제된다.
+//   - [x] 결제예상금액이 0원이 된다.
+// - [x] �� 버튼을 누르면 confirm 메시지가 보여진다.
+//   - [x] 확인을 누르면, 해당 상품이 삭제된다.
+// - [x] 체크된 상품 개수에 따라 주문하기 버튼 내부에 수량이 변경된다.
+// - [x] 주문하기 버튼을 누르면, confirm 메시지가 보여진다.
+//   - [x] 확인을 누르면, 주문/결제 페이지로 이동한다.
+//   - [x] 확인을 누르면, 장바구니에서 선택된 상품들이 삭제된다.
+//   - [x] 확인을 누르면, 체크된 상품들을 데이터베이스에서 제거한다.
+// - [x] 주문할 상품이 0개이면 버튼이 비활성화된다.
+/*
+const productInfo = {
+  id,
+  product: {},
+  quantity,
+  checked
+  ==>
+  id,
+  price,
+  name,
+  imageUrl,
+  quantity
+}
+*/
+const CartPage = () => {
+  const [cart, setCart] = useState([]);
+  const navigate = useNavigate();
+
+  const updateCart = async() => {
+    await API.getProduct("/carts")
+      .then(res => res.data)
+      .then(datas => datas.map(data => {
+        const {id, product} = data;
```

Comment:
```suggestion
      .then(datas => datas.map({id, product} => {
```
이렇게는 어떨까요? data를 뒤에서 사용하고 있는 것 같지 않고, datas에서 map을 해줬기 때문에 충분히 의미 전달도 되는 것 같아요. 꼭 data라는 것을 만들지 않아도 괜찮을 것 같아요!

Reply:
저렇게 수정하는게 나은 것 같네요! 감사합니다